firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -556,24 +597,34 @@ public void beginRealtimeHttpStream() {
                 }
               }
 
+              // Reset parameters.
+              httpURLConnection = null;
+              configAutoFetch = null;
+
               return Tasks.forResult(null);
             });
   }
 
-  // Pauses Http stream listening
-  public void closeRealtimeHttpStream(HttpURLConnection httpURLConnection) {
-    if (httpURLConnection != null) {
-      httpURLConnection.disconnect();
+  private void closeHttpConnectionInputStream(InputStream stream) {
+    if (stream == null) {
```

Comment:
nit: `inputStream` so it's obvious what this is

Reply:
updated